### ISSUE TITLE
Weather generator loading the nether

### ISSUE THREAD

**Issue by phyne** [OreCruncher/DynamicSurroundings#377]
Dynamic Surroundings 1.12.2-3.4.9.15
Forge 2795

`[19:21:17] [Server thread/INFO] [FML]: Loading dimension -1 (world) (net.minecraft.server.dedicated.DedicatedServer@7c461af3)
[19:21:17] [Server thread/INFO] [dsurround]: Creating NETHER weather generator for dimension [the_nether]
[19:21:21] [Server thread/WARN] [net.minecraft.server.MinecraftServer]: Can't keep up! Did the system time change, or is the server overloaded? Running 4055ms behind, skipping 81 tick(s)
[19:21:21] [Server thread/WARN] [net.minecraft.network.NetHandlerPlayServer]: Swampfox moved too quickly! -4.709103064659303,1.766367993957516,8.749784695814128
[19:21:26] [Server thread/INFO] [FML]: Unloading dimension -1`

It seems the nether is being loaded and unloaded routinely by the weather generator. I'm not running a powerhouse of a server and this is creating unnecessary load, not unlike a player constantly entering and leaving the nether (I have confirmed no players are going into the nether when this is happening). Is this supposed to be happening?

I'm unsure if other mods have anything to do with it, but I'll mention that I'm using NetherEx and BoP which add/change stuff in the nether.

**Comment by OreCruncher** [comment]
I don’t think it is DS doing this.  I will not be able to check until I get back into town (a week or so)

**Comment by OreCruncher** [comment]
Running single player I see Dim -1 being loaded, and then unloaded by Forge.  It's not DS.  The message about the weather generator is as a result of the dimension being loaded and firing a world load event.

**Comment by phyne** [comment]
So an unmodded forge instance loads and unloads dimensions periodically on its own?

**Comment by OreCruncher** [comment]
Honestly I have not looked at  an unmodded forge install for a long long time.  What I do know is that DS does not trigger world loads - it just responds via the world load/unload events whenever Forge fires them.

**Comment by phyne** [comment]
Well thanks for the input. Obviously not an issue with your mod (which of course is great btw). Sorry for cluttering the issues.
